cdap-ui/Gulpfile.js

```diff
@@ -289,6 +288,7 @@ gulp.task('watch:js:app:babel', function() {
     .pipe(plug.babel({
       presets: ['es2015']
     }))
+    .pipe(plug.ngAnnotate())
```

Comment:
No, best practice is to move ngAnnotate after Babel, then you can do ngInject on constructor. We don't have to manually add $inject to classes anymore